source/includes/_sequences.md

```diff
@@ -10,38 +15,54 @@ What is a sequence?
     "name": "Some series",
     "prefix": "XX",
     "padding": 7,
-    "suffix": "Y",
-    "ownerId": "db0db654-1694-4535-ae91-4506a206e4c2"
+    "suffix": "Y"
 }
 ```
 
-Information about the code object
+The object has three properties to create the terms of its sequence, `prefix`,
+`padding` and `suffix`. All the terms will share the same `prefix` and
+`suffix`, `padding` tells how many digits are set (at most) between those fixed
+values. Those digits represent a numerical value that always starts with one.
+Independently of the actual amount of digits of that number, the `padding`
+ensures the size of that portion of the entry's value, filling the gap with
+leading zeros.
 
 ## The Entry object
 
 ```json
 {
     "id": "550d89db-2b53-476c-a011-f796e196b053",
-    "codeId": "d4cfb49f-035b-46f0-b7b4-ec733aee2d00",
+    "code_id": "d4cfb49f-035b-46f0-b7b4-ec733aee2d00",
     "idx": 186,
-    "value": "XX0000186Y"
+    "value": "XX0000186Y",
+    "prev_id": "..."
```

Comment:
Does the user need to know the previous one or is this knowledge just an implementation detail that we use to build other entries?

Reply:
Good questions. I think the `code_id` and `prev_id` are more implementation details that the client probably doesn't need to know about. They probably also don't need the `idx` field. The `meta` information and signatures however could be useful for validation purposes.

Reply:
I think the `prev_id` might be useful. As a user, if I fetch all the entries, I would like to be able to order the sequence. Or we can guarantee the entries order.